libs/shared/tsconfig.json

```diff
@@ -4,15 +4,7 @@
     "rootDir": "./src",
     "baseUrl": "./src",
     "outDir": "./build/",
-    "tsBuildInfoFile": "./build/.buildinfo",
-    "plugins": [
-      {
-        "transform": "@formatjs/ts-transformer",
-        "import": "transform",
-        "type": "config",
-        "overrideIdFn": "[sha512:contenthash:base64:6]"
-      }
-    ]
```

Comment:
This was added in admin and removed here. Is it normal?

Reply:
Yeah, that's consistent with the rest of the projects... (I think).

It kinda makes no difference - VSCode-version TS will ignore it, and the build will get it either way.  But because VS-Code version of TS ignores these instructions, it seems tidier to have them just in the build file so it's clear where they are applied.